qemu.go

```diff
@@ -87,7 +87,7 @@ var supportedQemuMachines = []ciaoQemu.Machine{
 	},
 	{
 		Type:         QemuQ35,
-		Acceleration: "kvm,kernel_irqchip,nvdimm,nosmm,nosmbus,nosata,nopit,nofw",
+		Acceleration: "kvm,kernel_irqchip,nvdimm",
```

Comment:
you should factorize this as this is used for all qemu machine types.

Reply:
done